.eslintrc

```diff
@@ -33,6 +33,11 @@
     "react"
   ],
   "rules": {
+    "import/no-absolute-path": 0,
```

Comment:
Are these setting os any value in this PR, are they directly related to Digital products, I tend to believe you should never include as part of your commits, code changes that have nothing to do with your task as shown in the PR header